lldb/tools/lldb-dap/Handler/RequestHandler.h

```diff
@@ -394,6 +394,13 @@ class TestGetTargetBreakpointsRequestHandler : public LegacyRequestHandler {
   void operator()(const llvm::json::Object &request) const override;
 };
 
+class WriteMemoryRequestHandler : public LegacyRequestHandler {
+public:
+  using LegacyRequestHandler::LegacyRequestHandler;
+  static llvm::StringLiteral getCommand() { return "writeMemory"; }
```

Comment:
recently renamed in https://github.com/llvm/llvm-project/commit/1028ea9e2641341436555347fb65d6f2eafcfbdd

```suggestion
  static llvm::StringLiteral GetCommand() { return "writeMemory"; }
```